test/jobs/retrigger_test.js

```diff
@@ -15,11 +15,11 @@ suite('jobs/retrigger', function() {
 
     suite('duplicateTaskInTaskGroup', function() {
       test("duplicating a subgraph generates the right calls to queue.createTask", async function() {
-        fakeQueue.addTask('build', {payload: 'build', dependencies: ['other1'], taskGroupId: 'tgid'});
-        fakeQueue.addTask('test1', {payload: 'test1', dependencies: ['build', 'other2']});
-        fakeQueue.addTask('test2', {payload: 'test2', dependencies: ['build', 'test1']});
-        fakeQueue.addTask('sign', {payload: 'sign', dependencies: ['test1', 'test2', 'other3']});
-        let res = await job.duplicateTaskInTaskGroup('try', fakeQueue, 'build');
+        fakeQueue.addTask('buildId', {payload: 'build', dependencies: ['other1Id'], taskGroupId: 'tgid'});
+        fakeQueue.addTask('test1Id', {payload: 'test1', dependencies: ['buildId', 'other2Id']});
+        fakeQueue.addTask('test2Id', {payload: 'test2', dependencies: ['buildId', 'test1Id']});
+        fakeQueue.addTask('signId', {payload: 'sign', dependencies: ['test1Id', 'test2Id', 'other3Id']});
+        let res = await job.duplicateTaskInTaskGroup('try', fakeQueue, 'buildId');
```

Comment:
Is this just adding `Id` to each name, or is there something more subtle going on here?